.travis.yml

```diff
@@ -3,17 +3,17 @@
 #
 
 env:
-  - BUILD_TYPE=latexmk   # build using latexmk
+  - BUILD_TYPE=latexmk   # build using latexmk, also check for overfull hboxes
   - BUILD_TYPE=make      # build using Makefile
-  - BUILD_TYPE=manual    # build manually
-  - BUILD_TYPE=complete  # build manually and regenerate figures, grammer, and cross-references
+  - BUILD_TYPE=complete  # build manually and regenerate figures, grammar, and cross-references
   - BUILD_TYPE=check-whitespace  # check for whitespace at the ends of lines
 
 script:
   # Build std.pdf
   - pushd source
   - if [ "$BUILD_TYPE" = "latexmk" ]; then
       latexmk -pdf std;
+      ! grep -Fe "Overfull \hbox" std.log;
```

Comment:
Need to escape the backslash with another backslash.

Reply:
Done, thanks.